docs/src/main/sphinx/release/release-393.md

```diff
@@ -3,6 +3,13 @@
 ## General
 
 * Add support for {doc}`/sql/merge`. ({issue}`7933`)
+* Improve performance of highly selective `LIMIT` queries by making them finish
+  as soon as the required number of rows is produced. ({issue}`13616`)
+* Prevent inserting expressions that evaluate to null into non-nullable columns. ({issue}`13462`)
+* Simplify query logs by moving timeline messages to the debug log level. ({issue}`13668`)
+* Fix potential query stalling on bucketed tables. ({issue}`13655`)
```

Comment:
Is this a general thing? Or should that go into the affected connectors only?

Reply:
PR said it was a change to the core, so I imagine it's here.

Reply:
While Trino itself provides bucketed tables only in "some" connectors, it's possible for other connectors to provide bucketed tables. And the fix is in the engine instead of a specific connector.